src/services/inlayHints.ts

```diff
@@ -65,7 +65,7 @@ namespace ts.InlayHints {
             }
             else {
                 if (preferences.includeInlayFunctionParameterTypeHints && isFunctionLikeDeclaration(node) && hasContextSensitiveParameters(node)) {
-                    visitFunctionLikeForParameterType(node);
+                    visitFunctionLikeForParameterType(node, preferences.includeInlayFunctionLikeReturnTypeHints);

```

Comment:
@DanielRosenwasser  @Kingwl  Should we introduce a new preference like `includeInlayArrowFunctionAutoWrap` to control this feature ?